### PR TITLE
PX4 Vision Kit updates

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates the vision kit docs following my recent round of testing.
1. It now suggests you calibrate (if needed) via wifi. This did not work for me previously, but it does in this build.
2. With supplied USB stick boots in about 1 minute until the structure core shows a light. This compares to 2.5 mins using the one I had previously. I've updated docs throughout to say that - but also note that other cards may be slower.
3. Firmware update and airframe selection now work out of the box, so I updated the section on development to say that custom firmware is fine.
4. The cover on the upcore power supply is now there. I moved the warning down to the bottom and just changed it to say "don't take off the cover". I figure this is safer for people who already have one. 
5. Added a link from the Path Planning Interface doc to "advertise this".

### CODE REVIEW

**Comment by hamishwillee** [comment]
PS, I don't know why I have all those irritating merge commits. Ignore for now.

File: en/complete_vehicles/px4_vision_kit.md [hunk 1]
```diff
@@ -210,7 +211,7 @@ To flash the USB image to the *UP Core*:
 ### Boot the Companion Computer {#boot_mission_computer}
 
 First insert the provided USB2.0 stick into the *UP Core* port labeled `USB1`, and then power the vehicle using a 4S battery.
-The avoidance system should start within about 2.5 minutes.
+The avoidance system should start within about 1 minute (though this does depend on the USB stick supplied).
```
Comment (Jaeyoung-Lim) [inline en/complete_vehicles/px4_vision_kit.md]: Was this number from your experience? Could you check if your usb-stick as usb-2.0?

Comment (hamishwillee) [reply en/complete_vehicles/px4_vision_kit.md]: @Jaeyoung-Lim Yes. This was the time from power until the lasery thingy showed on the camera. It is a USB2.0 - SDCZ50-032G (Cruzer Blade 32GB).

This is the stick that came with the vehicle.

Comment (hamishwillee) [reply en/complete_vehicles/px4_vision_kit.md]: This otherwise OK?

**Comment by hamishwillee** [comment]
@Jaeyoung-Lim @jinger26 I am going to merge this as "no worse". I am assuming you still have no concerns with https://github.com/PX4/px4_user_guide/pull/700#discussion_r399081244

**Pull request merged**